examples/lb3-application/src/index.ts

```diff
@@ -4,16 +4,13 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {ApplicationConfig} from '@loopback/core';
-import {CoffeeShopApplication} from './application';
+import {ExpressServer} from './server';
 
-export async function main(options: ApplicationConfig = {}) {
-  const app = new CoffeeShopApplication(options);
-  await app.boot();
-  await app.start();
+export {ExpressServer};
 
-  const url = app.restServer.url;
-  console.log(`Server is running at ${url}`);
-  return app;
+export async function main(options: ApplicationConfig = {}) {
+  const server = new ExpressServer(options);
+  await server.boot();
+  await server.start();
+  console.log('Server is running at http://127.0.0.1:3000');
```

Comment:
should the host and ip values be hard-coded here?

Reply:
Those are defaults. We didn't include a config object to keep the code as minimal as possible.

Reply:
Can we use [`server.url`](https://github.com/strongloop/loopback-next/pull/4056/files#diff-8b388c43fa4dcdc667c3bd3d26d3496aR60)?

Reply:
Done.